From: Ole Streicher <olebole@debian.org>
Date: Sun, 30 Sep 2018 10:38:07 +0200
Subject: Add spdlog header

Closes: #909765
---
 cpp/sopt/logging.enabled.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cpp/sopt/logging.enabled.h b/cpp/sopt/logging.enabled.h
index b5eb6c1..7499cce 100644
--- a/cpp/sopt/logging.enabled.h
+++ b/cpp/sopt/logging.enabled.h
@@ -4,6 +4,7 @@
 #include "sopt/config.h"
 #include <spdlog/fmt/ostr.h>
 #include <spdlog/spdlog.h>
+#include <spdlog/sinks/stdout_sinks.h>
 #include "sopt/exception.h"
 
 namespace sopt {
