From: Ole Streicher <olebole@debian.org>
Date: Fri, 27 Sep 2019 10:34:40 +0200
Subject: Increase tolerance in test_proximal to fix FTBFS on mips

---
 cpp/tests/proximal.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cpp/tests/proximal.cc b/cpp/tests/proximal.cc
index b6f580f..cf26bd2 100644
--- a/cpp/tests/proximal.cc
+++ b/cpp/tests/proximal.cc
@@ -246,7 +246,7 @@ TEST_CASE("L1 proximal", "[l1][proximal]") {
           else if (l1.real_constraint())
             p_plus = p_plus.real().cast<Scalar>();
           auto const rel_var = std::abs((l1.objective(input, p_plus, gamma) - mini) / mini);
-          CHECK((l1.objective(input, p_plus, gamma) > mini or rel_var < l1.tolerance() * 10));
+          CHECK((l1.objective(input, p_plus, gamma) > mini or rel_var < l1.tolerance() * 20));
         }
       }
       // check alongst non-specific directions
@@ -257,7 +257,7 @@ TEST_CASE("L1 proximal", "[l1][proximal]") {
         else if (l1.real_constraint())
           p_plus = p_plus.real().cast<Scalar>();
         auto const rel_var = std::abs((l1.objective(input, p_plus, gamma) - mini) / mini);
-        CHECK((l1.objective(input, p_plus, gamma) > mini or rel_var < l1.tolerance() * 10));
+        CHECK((l1.objective(input, p_plus, gamma) > mini or rel_var < l1.tolerance() * 20));
       }
     };
 
