From: Mayeul d'Avezac <m.davezac@ucl.ac.uk>
Date: Wed, 12 Oct 2016 15:14:09 +0100
Subject: Logging api changed. Disabling colors for now.

Closes: #884219
---
 cpp/sopt/logging.enabled.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpp/sopt/logging.enabled.h b/cpp/sopt/logging.enabled.h
index e2837f6..9e45fde 100644
--- a/cpp/sopt/logging.enabled.h
+++ b/cpp/sopt/logging.enabled.h
@@ -13,7 +13,7 @@ void set_level(std::string const &level, std::string const &name = "");
 //! \brief Initializes a logger.
 //! \details Logger only exists as long as return is kept alive.
 inline std::shared_ptr<spdlog::logger> initialize(std::string const &name = "") {
-  auto const result = spdlog::stdout_logger_mt(default_logger_name() + name, color_logger());
+  auto const result = spdlog::stdout_logger_mt(default_logger_name() + name);
   set_level(default_logging_level(), name);
   return result;
 }
